OpenRA.Mods.D2k/Traits/Sandworm.cs

```diff
@@ -0,0 +1,130 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using System;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	class SandwormInfo : WandersInfo, Requires<MobileInfo>, Requires<RenderUnitInfo>, Requires<AttackBaseInfo>
+	{
+		[Desc("Time between rescanning for targets (in ticks).")]
+		public readonly int TargetRescanIntervalInTicks = 32;
+
+		[Desc("The radius in which the worm \"searches\" for targets.")]
+		public readonly int MaxSearchRadius = 32;
+
+		[Desc("The range at which the worm launches an attack regardless of noise levels.")]
+		public readonly int LaunchAttackRange = 3;
```

Comment:
`IgnoreNoiseAttackRange` is a bit better IMO as it clearly illustrates the purpose.